Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
We should use the 8-released tag if available, to avoid pre-releases

Reply:
Looks like the only tags available are "8.0" and "latest"
https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/ubi

Reply:
I think, this tag is not available for this image.
`Step 1/5 : FROM registry.access.redhat.com/ubi8/ubi-minimal:8-released
error parsing HTTP 404 response body: invalid character 'F' looking for beginning of value: "File not found.\""
`